bindings/jupyroot/python/JupyROOT/kernel/utils.py

```diff
@@ -62,10 +62,10 @@ def GetDisplayer(poller):
 class MagicLoader(object):
     '''Class to load JupyROOT Magics'''
     def __init__(self,kernel):
-         magics_path = os.path.dirname(__file__)+"/magics/*.py"
+         magics_path = os.path.dirname(__file__)+os.sep+"magics"+os.sep+"*.py"
```

Comment:
A more common way to write this would be

```python
os.path.join(os.path.dirname(__file__), "magics", "*.py")
```

Reply:
> A more common way to write this would be
> 
> ```python
> os.path.join(os.path.dirname(__file__), "magics", "*.py")
> ```

OK, I trust you